app/views/teams/users/_form.html.erb

```diff
@@ -0,0 +1,4 @@
+<fieldset class="usa-fieldset">
+  <%= form.input :email,
+      label: "<b>Email</b>".html_safe, label_html: { class: 'usa-label usa-input-required' }, input_html: { class: 'usa-input' }, required: true %>
```

Comment:
`usa-label` should already apply a bolded style to the label, so is it necessary to have the HTML ?

```suggestion
      label: 'Email', label_html: { class: 'usa-label usa-input-required' }, input_html: { class: 'usa-input' }, required: true %>
```